CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## Unreleased changes
 
+## 4.5.0
+
 - The node is now able to recover after crashes which leave only treestate or
   only blockstate usable.
 - Fix a memory leak that could occur in certain usage scenarios involving smart
   contracts.
+- Implementation of the the V2 GRPC API. This adds a number of new configuration
```

Comment:
Not sure 'implementation' is an important distinction here. 
I think something along the lines of: 
Support for a new GRPC API which uses typed proto definitions instead of json.

Reply:
Changed.